drop-in/src/main/java/com/adyen/checkout/dropin/SessionDropInService.kt

```diff
@@ -222,6 +223,27 @@ open class SessionDropInService : BaseDropInService(), SessionDropInServiceInter
         }
     }
 
+    override fun onRemoveStoredPaymentMethod(storedPaymentMethod: StoredPaymentMethod) {
+        launch {
+            val storedPaymentMethodId = storedPaymentMethod.id.orEmpty()
+            val result = sessionInteractor.removeStoredPaymentMethod(storedPaymentMethodId)
+
+            val serviceResult = when (result) {
+                SessionCallResult.RemoveStoredPaymentMethod.Successful ->
+                    RecurringDropInServiceResult.PaymentMethodRemoved(
+                        storedPaymentMethodId,
+                    )
+
+                is SessionCallResult.RemoveStoredPaymentMethod.Error -> RecurringDropInServiceResult.Error(
+                    errorDialog = ErrorDialog(),
```

Comment:
What if we explicitly set the error message here even if it will match the default one? I'm thinking it might be tough for someone to read this line and figure out why aren't we setting any error message before finding out it will fall back to the default one anyway